Node crashes when I stop it with cmd+c

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x459cf26]

goroutine 1 [running]:
github.com/mysteriumnetwork/node/services/openvpn/service.(*Manager).Stop(0xc0003590e0, 0xc0000df868, 0x1)
	/ext-go/1/src/github.com/mysteriumnetwork/node/services/openvpn/service/manager.go:98 +0x26
github.com/mysteriumnetwork/node/core/service.(*Pool).Stop(0xc0000a4080, 0xc00002a270, 0x24, 0x4a9ea00, 0xc000055010)
	/ext-go/1/src/github.com/mysteriumnetwork/node/core/service/pool.go:75 +0x16d
github.com/mysteriumnetwork/node/core/service.(*Pool).StopAll(0xc0000a4080, 0x10, 0xc0000df9a0)
	/ext-go/1/src/github.com/mysteriumnetwork/node/core/service/pool.go:86 +0xbb
github.com/mysteriumnetwork/node/core/service.(*Manager).Kill(0xc000097800, 0x4a096c0, 0xc0000dfa18)
	/ext-go/1/src/github.com/mysteriumnetwork/node/core/service/manager.go:143 +0x2f
github.com/mysteriumnetwork/node/cmd.(*Dependencies).Shutdown(0xc000001800, 0x0, 0x0)
	/ext-go/1/src/github.com/mysteriumnetwork/node/cmd/di.go:192 +0x375
github.com/mysteriumnetwork/node/cmd/commands/daemon.NewCommand.func2(0xc0001c2420, 0x47cb514, 0x19)
	/ext-go/1/src/github.com/mysteriumnetwork/node/cmd/commands/daemon/command.go:45 +0x2a
github.com/mysteriumnetwork/node/vendor/github.com/urfave/cli.Command.Run.func1(0xc0000dfd48, 0xc0001c2420, 0xc0000dfe78)
	/ext-go/1/src/github.com/mysteriumnetwork/node/vendor/github.com/urfave/cli/command.go:185 +0x3b
github.com/mysteriumnetwork/node/vendor/github.com/urfave/cli.Command.Run(0x47bb8d7, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0x47d27fb, 0x22, 0x0, ...)
	/ext-go/1/src/github.com/mysteriumnetwork/node/vendor/github.com/urfave/cli/command.go:215 +0x9b0
github.com/mysteriumnetwork/node/vendor/github.com/urfave/cli.(*App).Run(0xc000212000, 0xc0000a8000, 0x2, 0x2, 0x0, 0x0)
	/ext-go/1/src/github.com/mysteriumnetwork/node/vendor/github.com/urfave/cli/app.go:255 +0x687
main.main()
	/ext-go/1/src/github.com/mysteriumnetwork/node/cmd/mysterium_node/mysterium_node.go:54 +0xcd

--- a/services/openvpn/service/manager.go
+++ b/services/openvpn/service/manager.go
@@ -95,7 +95,9 @@ func (m *Manager) Serve(providerID identity.Identity) (err error) {
 
 // Stop stops service
 func (m *Manager) Stop() (err error) {
-	m.releasePorts()
+	if m.releasePorts != nil {
+		m.releasePorts()
+	}
 
 	if m.vpnServer != nil {
 		m.vpnServer.Stop()

--- a/services/openvpn/service/manager_test.go
+++ b/services/openvpn/service/manager_test.go
@@ -1,0 +1,30 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package service
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestManager_StopNotPanic(t *testing.T) {
+	m := Manager{}
+	err := m.Stop()
+	assert.NoError(t, err)
+}

